datasetIngestor/syncDataToFileserver_unix.go

```diff
@@ -1,39 +1,42 @@
+//go:build aix || darwin || dragonfly || freebsd || (js && wasm) || linux || nacl || netbsd || openbsd || solaris
 // +build aix darwin dragonfly freebsd js,wasm linux nacl netbsd openbsd solaris
 
 // very important: there must be an empty line after the build flag line .
 package datasetIngestor
 
 import (
 	"fmt"
-	"log"
-	"os"
+	"io"
 	"os/exec"
+	"regexp"
 	"strings"
+
 	version "github.com/mcuadros/go-version"
-	"regexp"
 )
 
 // functionality needed for "de-central" data
-func SyncDataToFileserver(datasetId string, user map[string]string, RSYNCServer string, sourceFolder string, absFileListing string) (err error) {
+// copies data from a local machine to a fileserver, uses RSync underneath
+func SyncLocalDataToFileserver(datasetId string, user map[string]string, RSYNCServer string, sourceFolder string, absFileListing string, cmdOutput io.Writer) (err error) {
```

Comment:
what's the reason for this extra arg, cmdOutput?

Reply:
is that used now?

Reply:
It is used for outputting the rsync progress to the stdout, when using the datasetIngest command with the copy flag

Reply:
but the file one is not used if I am not mistaken. Anyway, it doesn't harm